app/views/form/guidance/_finding_location.erb

```diff
@@ -0,0 +1,4 @@
+<%= govuk_details(summary_text: "What is a location?") do %>
+  <p class="govuk-body">A location is a postcode where supported housing is provided under a scheme. A scheme can have multiple locations, and a location can have multiple units at the same postcode.</p>
+  <p class="govuk-body"><%= govuk_link_to("Read more about how schemes and locations", scheme_changes_path) %></p>
```

Comment:
This isn't a full sentence - is there missing copy here?

Reply:
You make a very good point